debug/process/debugger.go

```diff
@@ -158,10 +160,31 @@ func dumpGoRoutines() {
 }
 
 func (debugger *processDebugger) changeLog() {
+	oldLogLevel := logger.GetLogLevelPattern()
+
 	errSetLogLevel := logger.SetLogLevel(debugger.debuggingLogLevel)
 	if errSetLogLevel != nil {
-		log.Error("debugger.changeLog: cannot change log level", "error", errSetLogLevel)
+		log.Error("processDebugger: cannot change log level", "error", errSetLogLevel)
+	}
```

Comment:
Missing return here?

Reply:
yes, we might return here as to retry the log change in about 4minutes